### ISSUE TITLE
💅 Possible false positive for `lint/suspicious/noFocusedTests` on functions named `fit`

### ISSUE THREAD

**Issue by hg** [biomejs/biome#6380]
### Environment information

```bash
CLI:
  Version:                      2.0.0
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           linux

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm-kitty
  JS_RUNTIME_VERSION:           v24.2.0
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         pnpm/10.12.1

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.json
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  false

Linter:
  JavaScript enabled:           unset
  JSON enabled:                 unset
  CSS enabled:                  unset
  GraphQL enabled:              unset
  Recommended:                  false
  Enabled rules:
    suspicious/noFocusedTests

Workspace:
  Open Documents:               0
```

### Rule name

lint/suspicious/noFocusedTests

### Playground link

https://biomejs.dev/playground/?lintRules=noFocusedTests&code=ZgB1AG4AYwB0AGkAbwBuACAAZgBpAHQAKAApACAAewAgAHIAZQB0AHUAcgBuACAANAAyADsAIAB9AAoAZgB1AG4AYwB0AGkAbwBuACAAbwBuAGwAeQAoACkAIAB7ACAAcgBlAHQAdQByAG4AIAAxADMAMwA3ADsAIAB9AAoACgBmAGkAdAAoACkAOwAKAG8AbgBsAHkAKAApADsACgAKAGMAbABhAHMAcwAgAEYAbwBvACAAewAKACAAIABmAGkAdAAoACkAIAB7ACAAcgBlAHQAdQByAG4AIAAnAG8AawAnADsAIAB9AAoAIAAgAHIAZQBuAGQAZQByACgAKQAgAHsAIAByAGUAdAB1AHIAbgAgAHQAaABpAHMALgBmAGkAdAAoACkAOwAgAH0ACgB9AA%3D%3D

### Expected result

Hi, I just recently started using Biome, so not 100% sure if this is a false-positive, or I'm doing something stupid, but the errors shown here:

<details>

<summary>click</summary>

```
main.tsx:4:1 [lint/suspicious/noFocusedTests](https://biomejs.dev/linter/rules/no-focused-tests)  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ⚠ Don't focus the test.
  
    2 │ function only() { return 1337; }
    3 │ 
  > 4 │ fit();
      │ ^^^
    5 │ only();
    6 │ 
  
  ℹ The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
  
  ℹ Consider removing 'only' to ensure all tests are executed.
  
  ℹ Unsafe fix: Remove focus from test.
  
     2  2 │   function only() { return 1337; }
     3  3 │   
     4    │ - fit();
        4 │ + it();
     5  5 │   only();
     6  6 │   
  
main.tsx:9:26 [lint/suspicious/noFocusedTests](https://biomejs.dev/linter/rules/no-focused-tests)  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ⚠ Don't focus the test.
  
     7 │ class Foo {
     8 │   fit() { return 'ok'; }
   > 9 │   render() { return this.fit(); }
       │                          ^^^
    10 │ }
  
  ℹ The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
  
  ℹ Consider removing 'only' to ensure all tests are executed.
  
  ℹ Unsafe fix: Remove focus from test.
  
     7  7 │   class Foo {
     8  8 │     fit() { return 'ok'; }
     9    │ - ··render()·{·return·this.fit();·}
        9 │ + ··render()·{·return·this.it();·}
    10 10 │   }
```

</details>

seem to have nothing to do with the code?

The test rules should be enabled globally just like any other rule if I'm not mistaken, and they mention `only` for functions named `fit`, and don't complain about the actual `only`.

Here's a minimal repro config:

```json
{
	"$schema": "https://biomejs.dev/schemas/2.0.0/schema.json",
	"files": {
		"ignoreUnknown": false
	},
	"linter": {
		"enabled": true,
		"rules": {
			"recommended": false,
			"suspicious": {
				"noFocusedTests": "error"
			}
		}
	}
}
```

Please also see the playground link. Thanks.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by wojtekmaj** [comment]
Also got this, surprisingly when updated Biome to 2.0 :) 

Even more minimal repro:

https://biomejs.dev/playground/?lintRules=noFocusedTests&code=ZgB1AG4AYwB0AGkAbwBuACAAZgBpAHQAKAApACAAewB9AAoACgBmAGkAdAAoACkA

**Comment by vladimir-ivanov** [comment]
so we just need to fix `only` to become `fit` in here? 

<img width="981" alt="Image" src="https://github.com/user-attachments/assets/47078900-b27c-4a69-b21c-b1f84e00fee3" />

**Comment by vladimir-ivanov** [comment]
it is on any "f" prefixed function, will address, thanks
<img width="905" alt="Image" src="https://github.com/user-attachments/assets/1a497fd4-892a-4d2d-ba1b-13af73ae2cfa" />

**Comment by hg** [comment]
I think it's a separate issue. The rule simply should not be triggered if I understand it correctly. Compare these two branches (they differ only in the version of biome used):

```
% git clone https://github.com/hg/biome-fit-rule -b biome194 biome194
% cd biome194
% pnpm i
% pnpm biome check
Checked 3 files in 5ms. No fixes applied.
```

```
% git clone https://github.com/hg/biome-fit-rule -b biome206 biome206
% cd biome206
% pnpm i
% pnpm biome check
index.jsx:4:1 lint/suspicious/noFocusedTests  FIXABLE  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Don't focus the test.
  
    2 │ 	return 42;
    3 │ }
  > 4 │ fit();

...
```
